File: Python/1099.py
n = int(input())
d = 0

for c in range(1 , n + 1):
    x = input().split()
    a, b, = x
    s=0
    a = int(a)
    b = int(b)

    if a > b:
        for d in range(int(b)+1, int(a)):
            if d % 2 != 0:
                s = s + d
    if a < b:
        for d in range(int(a)+1, int(b)):
            if d % 2 != 0:
                s = s + d
    if a == b:
        s = 0
    print(s)

File: CSharp/1099.cs
using System;
using System.Linq;


class URI {

    static void Main(string[] args) 
    {
        int n = int.Parse(Console.ReadLine());
        int[] x;
        int soma =0;

        for(int i = 0; i < n;i++)
        {
            x = Console.ReadLine().Split().Select(int.Parse).ToArray();

            Array.Sort(x);

            if(x[0]==x[1])
            {soma=0;}
            else
            {
                for(int j=x[0]+1;j<x[1];j++)
                {
                    if(j%2!=0)
                    {soma+=j;}
                }
            }
            Console.WriteLine(soma);
            soma=0;

        }
    }

}

